include/net/nrf_cloud_cell_pos.h

```diff
@@ -69,6 +69,17 @@ int nrf_cloud_cell_pos_request(const struct lte_lc_cells_info * const cells_inf,
 			       const bool request_loc);
 #endif /* CONFIG_NRF_CLOUD_MQTT */
 
+/**@brief Parses the cellular positioning response (REST and MQTT) from nRF Cloud.
+ *
+ * @param buf Pointer to data received from nRF Cloud.
+ * @param result Decoded location information.
+ * @retval 0 if successful. 1 if buf does not contain a cell pos response; continue
+ * processing by other handlers. Negative if the cell pos response is badly formatted
+ * or buf is NULL.
+ */
+int nrf_cloud_parse_cell_pos_response(const char *const buf,
```

Comment:
`nrf_cloud_cell_pos_process` is the public version... sorry i pointed you to the wrong function in my previous comment.

Reply:
Good grief.  It was *right there* under this one, after I moved it.

Reply:
Removing this commit... nothing to see here... move along.